tools/analysis/analysis_test.py

```diff
@@ -23,7 +23,9 @@ def testRunNotebook(self):
             # due to bash's limit of 128 chars for the shebang line
             # see: https://github.com/pypa/pip/issues/1773
             # see: https://www.in-ulm.de/~mascheck/various/shebang/#length
-            cmd = ['python', os.path.join(conda_env, 'bin', 'jupyter-nbconvert')]
+            python = os.path.join(conda_env, 'bin', 'python')
+            nbconvert = os.path.join(conda_env, 'bin', 'jupyter-nbconvert')
+            cmd = [python, nbconvert]
```

Comment:
Doesn't conda automatically ensure that binaries in the bin dir use the environment's python?  (I'm guessing you put this here because it doesn't, though...?  :-)

Reply:
It does that by using the shebang line, however this is broken on linux because the shebang line can only be something paltry like 128 chars. So we must invoke python directly.

Reply:
SGTM